pkg/services/dashboards/database/database_test.go

```diff
@@ -639,6 +639,41 @@ func TestIntegrationDashboard_Filter(t *testing.T) {
 	assert.Equal(t, dashB.ID, results[0].ID)
 }
 
+func TestGetExistingDashboardByTitleAndFolder(t *testing.T) {
+	sqlStore := db.InitTestDB(t)
+	cfg := setting.NewCfg()
+	cfg.IsFeatureToggleEnabled = func(key string) bool { return false }
+	quotaService := quotatest.New(false, nil)
+	dashboardStore, err := ProvideDashboardStore(sqlStore, cfg, testFeatureToggles, tagimpl.ProvideService(sqlStore, cfg), quotaService)
+	require.NoError(t, err)
+	insertTestDashboard(t, dashboardStore, "Apple", 1, 0, false)
+	t.Run("Should be able to get dashboard with existing name", func(t *testing.T) {
+		err = sqlStore.WithDbSession(context.Background(), func(sess *sqlstore.DBSession) error {
+			_, err = getExistingDashboardByTitleAndFolder(sess, &dashboards.Dashboard{Title: "Apple", OrgID: 1}, sqlStore.GetDialect(), false, false)
+			return err
+		})
+		require.Error(t, err)
+	})
+
+	t.Run("Should not be able to get dashboard with non-existing name", func(t *testing.T) {
+		err = sqlStore.WithDbSession(context.Background(), func(sess *sqlstore.DBSession) error {
+			_, err = getExistingDashboardByTitleAndFolder(sess, &dashboards.Dashboard{Title: "Beta"}, sqlStore.GetDialect(), false, false)
```

Comment:
```suggestion
			_, err = getExistingDashboardByTitleAndFolder(sess, &dashboards.Dashboard{Title: "Beta", OrgID: 1}, sqlStore.GetDialect(), false, false)
```
we want to test [this](https://github.com/grafana/grafana/blob/b80c20f80ba0d0de223d81e912c9fe57224c72c9/pkg/services/dashboards/database/database.go#L421) specific case (no error if no dashboard with this title exists) but we have to set the org as well otherwise the [query](https://github.com/grafana/grafana/blob/b80c20f80ba0d0de223d81e912c9fe57224c72c9/pkg/services/dashboards/database/database.go#L393) will not return any results regardless of the title we pass.

Reply:
"Returns no error when dashboard does not exist in root directory"